src/main/resources/bucketeer_messages.xml

```diff
@@ -62,7 +62,7 @@
   <entry key="BUCKETEER-051">Slack user '{}' uploaded CSV file '{}' to: {}</entry>
   <entry key="BUCKETEER-052">CSV contains an entry that is not a work or collection: {}</entry>
   <entry key="BUCKETEER-053">'{}' claims it's a collection but it contains a parent ARK and/or a file name</entry>
-  <entry key="BUCKETEER-054">'{}' claims it's a work but it doesn't have a parent ARK and/or a file name</entry>
+  <entry key="BUCKETEER-054">'{}' claims it's a work, but it doesn't have an image file on the file system</entry>
```

Comment:
"work" is a generic term, "image" is specific... if we start handling more than images in Bucketeer, this message could be confusing

Reply:
Thanks, deleting... those are from the previous, now obsolete code. I need to write something that scans project code for all the MessageCodes and makes a list of things to delete (things that aren't actually used).